README.md

```diff
@@ -70,6 +70,20 @@ If your EdgeX instance is not running on localhost, minimally you will need to r
 You can now use the CLI by entering `edgex-cli` anywhere on your machine provided your $GOBIN is on your $PATH.
 
 
+#### Interactive Mode
+
+Some commands leverage interactive-mode which opens an editor and allows you to provide information that would 
+normally be difficult with just command line arguments. For example, creating an Event requires a lot of information,
+also Events contain zero or more readings. Using interactive mode, you can easily create an Event with many readings and 
+customize each reading. You can choose the editor that is used by setting the environment variable `EDITOR`. The default
+editor is `Vi` and is used if no `EDITOR` is specified. Some examples of editors:
```

Comment:
Could we make the default based on whether the operating system is Linux or Windows. as opposed to always vi ?

Reply:
Very good point. What would be a good editor for Windows? Notepad?

Reply:
You got it, faithful Notepad!